packages/webdriverio/webdriverio-core.d.ts

```diff
@@ -389,6 +389,12 @@ declare namespace WebdriverIO {
         args: Array<string>
     }
 
+    type ViewMatcher = {
```

Comment:
Can there be a `class` property in data matcher selectors? If so it might make sense to merge both types.

Reply:
Yes, DataMatcher and ViewMatcher have the same object structure. I can combine the two in the just one Matcher type for TS. What do you think?

Reply:
Sounds good!